definitions/procedures/foreman_tasks/resume.rb

```diff
@@ -8,6 +8,8 @@ class Resume < ForemanMaintain::Procedure
 
     def run
       output << feature(:foreman_tasks).resume_task_using_hammer
+      puts '[ Waiting 30 seconds for resumed tasks to start. ]'
```

Comment:
Do we need square braces ? if not then those can be removed ? One more thing is, when we delete tasks after this, assuming it did not resume task count may change. I think we should inform user about that ?